ui-tests/truffle/migrations/2_deploy_contracts.js

```diff
@@ -1,9 +1,11 @@
 const SimpleStorage = artifacts.require("SimpleStorage");
 const TutorialToken = artifacts.require("TutorialToken");
 const ComplexStorage = artifacts.require("ComplexStorage");
+const Grue = artifacts.require("Grue");
```

Comment:
Are these contracts used?

Reply:
The first three contracts are used but we can remove Grue from the migrations and the contracts directory.